Update to new compilers

dbast opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

dbast committed:
Update to new compilers

r-rgl is now available.

dbast committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.2.14, and conda-forge-pinning 2019.03.14

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Using toolchain directly in this manner is deprecated.  Consider using the compilers outlined [here](https://conda-forge.org/docs/meta.html#compilers).

dbast committed:
Add workaround to make libGL available during build + test

Users shouldn't need to set LD_LIBRARY_PATH, libGL will be in some default
location on their system. This trick should only be needed during CI
testing since it's in a non-standard location.

How it is resloved on user side:
~/miniconda3/envs/r-rgl$ ldd -r lib/R/library/rgl/libs/rgl.so
	linux-vdso.so.1 (0x00007ffeb39cf000)
	libGLU.so.1 => /home/$USER/miniconda3/envs/r-rgl/lib/R/library/rgl/libs/../../../../libGLU.so.1 (0x00007f4760ea6000)
	libGL.so.1 => /usr/lib/x86_64-linux-gnu/libGL.so.1 (0x00007f4760ac9000)
        ...

dbast committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.3.0, and conda-forge-pinning 2019.03.17

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Using toolchain directly in this manner is deprecated.  Consider using the compilers outlined [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#compilers).
